src/Catty/PlayerEngine/Stage/Builder/StageBuilder.swift

```diff
@@ -39,10 +39,17 @@
 
         self.logger = stageLogger
 
+        if !project.header.landscapeMode {
```

Comment:
Could you please double check if that is necessary? When I create a project in landscape mode using Catroid (e.g., https://share.catrob.at/app/project/6c0dfc4c-dd62-11ea-a5c3-005056a36f47), then the height is smaller than the width, meaning that the coordinates are already switched.

Reply:
Hey @m-herold I checked it again. Yes the changes are needed, otherwise the Scene is presented like in the screenshot.
![IMG_44475E27B87A-1](https://user-images.githubusercontent.com/44603698/90344023-2a315580-e016-11ea-9059-959a40e90e08.jpeg)

Reply:
Thanks -- I see. Nevertheless, could you please provide me with a link to the project you created like shown in the screenshot?

Reply:
Of course, for example https://share.catrob.at/app/project/01475b14-e04c-11ea-a5c3-005056a36f47. This project is with the implementation in the _StageBuilder.swift_

Reply:
Could you please create the same project with Android? I think there the height and width should be reversed (leading to the above mentioned code-switch to be obsolete).

Reply:
That's the link to the Android Project https://share.catrob.at/app/project/811b16ea-e053-11ea-a5c3-005056a36f47

Reply:
Thanks for your input, I made your suggested changes!